### PR TITLE
Add feedback guide and team practices section

### PR DESCRIPTION
**Pull request by haroldcampbell**
This pull request adds a new section to the Team Operations chapter of the Team Compass. It introduce team-practices sub-sections and feedback as the first guide

### CODE REVIEW

**Review by choldgraf — APPROVED** [review]
This looks good to me, I made two quick changes:

1. Added this chapter to the `toctree` of the `index` page for team operations. That way the parent page shows up in the sidebar
2. Updated some of the markdown headers so that we only have a single top-level header `#` and all other headers start at `##`.

I also [recorded a short video showing how I arrived at seeing (1) was an issue here](https://2i2c.slack.com/archives/C06G5FAAT63/p1712088829346289).

I don't think either is controversial, so will merge this!

File: operations/team-practices/feedback.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+2 – [The SCARF model](https://www.mindtools.com/akswgc0/david-rocks-scarf-model)
+3 – [Value judgements](https://laurensergy.com/value-judgements/)
+4 – Radical Candor ([4 mins video](https://www.youtube.com/watch?v=rFgu0nOHCcE))
+5 – [The 3 Levels of Listening](https://medium.com/swlh/strategic-leadership-the-3-levels-of-listening-e3f0c27f8d01)
+6 – Crucial Conversations ([video summary](https://www.youtube.com/watch?v=Q2yG142cyNg)), ([book on Amazon](https://www.amazon.com/Crucial-Conversations-Talking-Stakes-Second/dp/0071771328))
+7 – [GE’s Real-Time Performance Development](https://hbr.org/2015/08/ges-real-time-performance-development) introduces Continue/Consider
+8 – [Impact Feedback](https://vividbreeze.com/impact-feedback/)
+9 – [Primed to Perform](https://yantougas.com/2016/02/15/blame/) REAP explanation
+10 – Rise Model ([2 mins video](https://www.youtube.com/watch?v=3cn306Fo_IU)) [www.risemodel.com](https://www.risemodel.com)
+11 – [The Shit Sandwich and Other Terrible Ways to Give Feedback](https://blog.idonethis.com/sandwich-feedback-performance-management/#:~:text=The%20main%20idea%20behind%20the,results%20in%20the%20exact%20opposite).
```
Comment (choldgraf) [inline operations/team-practices/feedback.md:247]:
```suggestion

1. [Ladder of inference](https://thesystemsthinker.com/the-ladder-of-inference/)
2. [The SCARF model](https://www.mindtools.com/akswgc0/david-rocks-scarf-model)
3. [Value judgements](https://laurensergy.com/value-judgements/)
4. Radical Candor ([4 mins video](https://www.youtube.com/watch?v=rFgu0nOHCcE))
5. [The 3 Levels of Listening](https://medium.com/swlh/strategic-leadership-the-3-levels-of-listening-e3f0c27f8d01)
6. Crucial Conversations ([video summary](https://www.youtube.com/watch?v=Q2yG142cyNg)), ([book on Amazon](https://www.amazon.com/Crucial-Conversations-Talking-Stakes-Second/dp/0071771328))
7. [GE’s Real-Time Performance Development](https://hbr.org/2015/08/ges-real-time-performance-development) introduces Continue/Consider
8. [Impact Feedback](https://vividbreeze.com/impact-feedback/)
9. [Primed to Perform](https://yantougas.com/2016/02/15/blame/) REAP explanation
10. Rise Model ([2 mins video](https://www.youtube.com/watch?v=3cn306Fo_IU)) [www.risemodel.com](https://www.risemodel.com)
11. [The Shit Sandwich and Other Terrible Ways to Give Feedback](https://blog.idonethis.com/sandwich-feedback-performance-management/#:~:text=The%20main%20idea%20behind%20the,results%20in%20the%20exact%20opposite).
```

File: operations/team-practices/feedback.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+  - It’s from a harasser, bully, gossip, etc.
+  - It’s being done publicly
+  - It’s not being done to help them grow
+
+#### Deferring feedback with “Not now” 
+
+“Not now” isn’t a no, and it’s also a reasonable thing to say - if you know you’ll have a hard time receiving feedback right now, suggest another, better time. 
+
+For instance:
+  Molly (Giver): Tod, may I give you some feedback based on…
```
Comment (yuvipanda) [inline operations/team-practices/feedback.md:103]:
```suggestion
  Molly (Giver): Todd, may I give you some feedback based on…
```

File: operations/team-practices/feedback.md [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+  _**Tod (Receiver): I am a bit overwhelmed and not in a place to receive feedback right now!**_
```
Comment (yuvipanda) [inline operations/team-practices/feedback.md:104]:
```suggestion
  _**Todd (Receiver): I am a bit overwhelmed and not in a place to receive feedback right now!**_
```

File: operations/team-practices/feedback.md [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+  Molly: Thank you for sharing with me how you feel I understand. Because this is important, is there a time that you could recommend?
+  Tod: Thanks for understanding, may we try again next week?...
```
Comment (yuvipanda) [inline operations/team-practices/feedback.md:106]:
```suggestion
  Todd: Thanks for understanding, may we try again next week?...
```

**Review by yuvipanda — APPROVED** [review]
This was extremely helpful for me already! I have one suggestion :)

**Pull request merged**